certified-connectors/Softools/apiDefinition.swagger.json

```diff
@@ -0,0 +1,699 @@
+        ],
+        "responses": {
+          "200": {
+            "description": "Response also includes all record data with field identifier as key",
+            "schema": {
+              "$ref": "#/definitions/Message"
+            }
+          },
+          "401": {
+            "description": "Unauthorized",
+            "examples": {
+              "application/json": "{\n  \"Message\": \"Authorization has been denied for this request.\"\n}"
+            }
+          }
+        }
+      }
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.softools.net"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.softools.net/privacy-policy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Business Intelligence;Data"
+    }
+  ],
+  "definitions": {
+    "Message": {
+      "type": "object",
+      "properties": {
+        "Message": {
+          "type": "string"
+        }
+      },
+      "example": {
+        "Message": "string"
+      }
+    },
+    "Records": {
+      "type": "object"
+    },
+    "basic-fields-response": {
+      "type": "object",
+      "properties": {
+        "identifier": {
+          "type": "string"
+        },
+        "Type": {
+          "type": "integer"
+        }
+      }
+    },
+    "List-record-response": {
+      "type": "array",
+      "items": {}
+    },
+    "Record-Response": {
+      "type": "object",
+      "properties": {
+        "_id": {
+          "type": "string"
+        },
+        "AppIdentifier": {
+          "type": "string"
+        },
+        "CreatedByUserId": {
+          "type": "string"
+        },
+        "CreatedByUser": {
+          "type": "string"
+        },
+        "CreatedDate": {
+          "type": "object",
+          "properties": {
+            "$date": {
+              "type": "integer"
+            }
+          }
+        },
+        "UpdatedByUserId": {
+          "type": "string"
+        },
+        "UpdatedByUser": {
+          "type": "string"
+        },
+        "UpdatedDate": {
+          "type": "object",
+          "properties": {
+            "$date": {
+              "type": "integer"
+            }
+          }
+        },
+        "Identifier": {
+          "type": "string"
+        },
+        "QuickFilterSearchText": {
+          "type": "string"
+        },
+        "Id": {
+          "type": "string"
+        },
+        "EditableAccessForUser": {
+          "type": "boolean"
+        }
+      }
+    },
+    "Report-Field": {
+      "type": "array",
+      "items": {}
+    },
+    "Update-record": {
+      "type": "object"
+    }
+  },
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "APIKey": {
+      "type": "apiKey",
+      "in": "header",
+      "name": "apikey"
+    },
+    "Tenant": {
+      "type": "apiKey",
+      "in": "header",
+      "name": "tenant"
```

Comment:
I would like to set the `tenant` for all requests. This is the `AppId`. In the swagger docs it a Multiple Key example, is this supported in connectors? https://swagger.io/docs/specification/authentication/api-keys/

Reply:
Looks like this is not supported? https://powerusers.microsoft.com/t5/Connecting-To-Data/Multiple-API-Keys/td-p/402250 

For now the user will need to set the `tenant` header with each request.